Pipfile.lock

```diff
@@ -403,14 +403,6 @@
             "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
             "version": "==2.10"
         },
-        "importlib-metadata": {
-            "hashes": [
-                "sha256:c9db46394197244adf2f0b08ec5bc3cf16757e9590b02af1fca085c16c0d600a",
-                "sha256:d2d46ef77ffc85cbf7dac7e81dd663fde71c45326131bea8033b9bad42268ebe"
-            ],
-            "index": "pypi",
-            "version": "==3.10.0"
-        },
```

Comment:
I don't think that this file should be changed here since the user will run the needed install commands anyways. It's fine if you want to keep the changes but it's good practice to keep the pull request focused on the changes you're adding.